src/extension/options-page/Router/Contacts.tsx

```diff
@@ -55,16 +67,45 @@ export default function DashboardContactsRouter() {
             </Typography>
             {/* without flex: 1, the auto resize cannot resize to the max height it need. */}
             <section style={{ flex: 1 }}>
-                <AutoResize>
-                    {(size) => (
-                        <FixedSizeList {...size} itemSize={64} itemCount={contacts.length}>
-                            {({ index, style }) => (
-                                <ContactLine style={style} key={index} contact={contacts[index]}></ContactLine>
-                            )}
-                        </FixedSizeList>
-                    )}
-                </AutoResize>
+                <Suspense fallback={contactLineFallback}>
+                    <ContactsList query={search === '' ? void 0 : search} />
```

Comment:
Handle `undefined` in `<ContactsList  />`?

Reply:
I think there is no difference